### PR TITLE
Updating design guidelines

### PR DESCRIPTION
**Pull request by rachael-phillips**
Creating pull request for updating design guidelines

### CODE REVIEW

File: packages/patternfly-4/content/get-started/gettingstarted-design.md [hunk 1]
```diff
@@ -1,42 +1,105 @@
 ---
 path: "/get-started/designers"
 ---
+Whether you are converting from a PatternFly 3 application or starting fresh with the latest and greatest PatternFly design system, this document is intended to help provide designers with information regarding resources and support available.
+
 # Design with PatternFly
-Before you begin, check out our [overview of PatternFly](/get-started/about) to get acquainted with the basic elements of the design system.
+* [Getting the design kit](#getting-the-design-kit)
+* [Using the design kit](#using-thed-esign-kit)
```
Comment (mcarrano) [inline packages/patternfly-4/content/get-started/gettingstarted-design.md]: Looks like you have a typo in this link?

**Comment by rachael-phillips** [comment]
@mcarrano the images should be there now - let me know if you are seeing any issues

**Comment by mcarrano** [comment]
@jschuler can you take a look at this? It is trying to update the Get Started page for designers. Looks like merging is blocked. I don't really understand what these conflicts are. Thanks.

**Comment by jschuler** [comment]
Closing in favor of https://github.com/patternfly/patternfly-org/pull/1480

**Pull request closed without merging**